src/ui/ducks/identities.ts

```diff
@@ -12,21 +12,40 @@ import { useAppSelector } from "./hooks";
 
 export interface IdentitiesState {
   identities: IdentityData[];
+  hostIdentities: IdentityData[];
+  randomIdentities: IdentityData[];
   operations: Operation[];
   requestPending: boolean;
   selected: SelectedIdentity; // This aim to be a short-term solution to the integration with Zkitter
+  selectedToConnect: SelectedIdentity;
+  connected: SelectedIdentity;
+  notReadyToConnect: boolean;
```

Comment:
This can be removed and used on ui side without redux. It's just a validation when to enable button.